src/adapters/node.js

```diff
@@ -1,36 +1,53 @@
-// Generated by CoffeeScript 1.9.3
-(function() {
+(function () {
     var request = require('request');
     var mkFhir = require('../fhir');
     var Q = require('q');
 
     var adapter = {
         defer: Q.defer,
-        http: function(args) {
+        http: function (args) {
             var deff = Q.defer();
-            args.body = args.data;
```

Comment:
Can you briefly explain why it is okay to delete this line? This same logic appears in other adapters and if it is incorrect it would be good to remove across the board.